players/g1_player.py

```diff
@@ -44,7 +44,7 @@ def spread_points(current_point, angles: np.array, distance, reverse) -> np.arra
 def splash_zone(distance: float, angle: float, conf: float, skill: int, current_point: Tuple[float, float], in_sandtrap: bool) -> np.array:
     conf_points = np.linspace(1 - conf, conf, 5)
     scale = 1.1
-    if in_sandtrap: 
+    if in_sandtrap:
         # the distance rating is halved, and the standard deviations for the angle and distance distributions are doubled
         distances = np.vectorize(standard_ppf)(
             conf_points) * (distance / skill) * 2
```

Comment:
Did we miss + distance or + 2 * distance here?

When the standard deviation is not doubled, the formula for distance is:

```python
distances = np.vectorize(standard_ppf)(conf_points) * (distance / skill) + distance
```

Reply:
There should be a distance / 2

Reply:
It's weird, maybe I deleted it by mistake

Reply:
I'm not exactly sure what `distance / skill` represents, but based on the relationship that the distances from the sand trap should be doubled, I think the formulas should be:

```python
distances = np.vectorize(standard_ppf)(conf_points) * 2 * (distance / skill) + distance   # from sand trap
distances = np.vectorize(standard_ppf)(conf_points) *  (distance / skill) + distance  # from grass
```

What do you think?

Reply:
I agree with you, and I already made some changes just now, the formula is the same as what you provided.